### PR TITLE
Monitoring

### PR DESCRIPTION
**Pull request by jmdobry**

### CODE REVIEW

**Comment by jmdobry** [comment]
@waprin PTAL

File: monitoring/quickstart.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      // Limit results to the last 10 minutes
+      seconds: (Date.now() / 1000) - (60 * 10)
+    },
+    endTime: {
+      seconds: Date.now() / 1000
+    }
+  }
+};
+
+// Writes time series data
```
Comment (waprin) [inline monitoring/quickstart.js]: Incorrect comment?

File: monitoring/quickstart.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START monitoring_quickstart]
+// Imports the Google Cloud client library
```
Comment (waprin) [inline monitoring/quickstart.js]: I think the biggest use case for the API in code (as opposed to ops-y stuff done in gcloud or stuff done in fluentd) is writing custom metric so we might want to consider making that the quickstart.

Comment (jmdobry) [inline monitoring/quickstart.js]: Done

File: monitoring/quickstart.js (hunk 1 above)
Comment (jmdobry) [inline monitoring/quickstart.js]: Fixed

**Comment by jmdobry** [comment]
@waprin Ready for final review.

**Comment by waprin** [comment]
LGTM

**Pull request merged**